ush/gfs_bufr.sh

```diff
@@ -100,17 +100,16 @@ ${NLN} "${COMIN_ATMOS_HISTORY}/${RUN}.${cycle}.sfc.f${fhr_p}.${atmfm}" "flxf${fh
 ${NLN} "${PARMgfs}/product/bufr_gfs_${CLASS}.tbl" fort.1
 ${NLN} "${STNLIST:-${PARMgfs}/product/bufr_stalist.meteo.gfs}" fort.8
 
-
 #------------------------------------------------------------------
 "${EXECgfs}/${pgm}" < gfsparm > "out_gfs_bufr_${fhr}"
 
 export err=$?
 
 if [[ ${err} -ne 0 ]]; then
-   echo "WARNING GFS postsnd job error, Please check files "
-   echo "${COMIN_ATMOS_HISTORY}/${RUN}.${cycle}.atm.f${fhr}.${atmfm}"
-   echo "${COMIN_ATMOS_HISTORY}/${RUN}.${cycle}.sfc.f${fhr}.${atmfm}"
-   exit "${err}"
+    echo "FATAL ERROR: GFS postsnd job error, Please check files "
+    echo "${COMIN_ATMOS_HISTORY}/${RUN}.${cycle}.atm.f${fhr}.${atmfm}"
+    echo "${COMIN_ATMOS_HISTORY}/${RUN}.${cycle}.sfc.f${fhr}.${atmfm}"
+    err_exit
```

Comment:
I'm not sure if this should be a fatal error or not. What do you think @BoCui-NOAA?